sacc/data_types.py

```diff
@@ -298,13 +297,6 @@ def __init__(self, data_type, tracers, value,
                                      f"{data_type} "
                                      "(ignore_missing_tags=False)")
 
-        # We encourage people to use existing type names, and issue a
-        # warning if they do not to prod them in the right direction.
-        if data_type not in standard_types:
-            warnings.warn(f"Unknown data_type value {data_type}. "
-                          "If possible use a pre-defined type, or "
-                          "add to the list.")
-
```

Comment:
I like this change, but just checking that it's not been removed in error.

Reply:
yeah, it was deliberate. It's just been annoying unnecessarily.